### PR TITLE
Refacto international tests

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Refacto international tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | No tests needed
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25652)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/11_international/01_localization/02_languages/02_CRUDLanguage.js [hunk 1]
```diff
@@ -159,18 +171,29 @@ describe('CRUD language', async () => {
       await expect(numberOfLanguagesAfterReset).to.be.equal(numberOfLanguages + 1);
     });
 
-    it(`should go to FO and check that '${editLanguageData.name}' do not exist`, async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'checkUpdatedLanguageFO', baseContext);
+    it('should go to FO', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToFo2', baseContext);
 
-      // View my shop and init pages
+      // View my shop and int pages
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/01_localization/02_languages/02_CRUDLanguage.js]:
```suggestion
      // View my shop and init pages
```

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/05_sortAndPagination.js [hunk 2]
```diff
@@ -12,14 +12,13 @@ const taxesPage = require('@pages/BO/international/taxes');
 // Import test context
 const testContext = require('@utils/testContext');
 
-const baseContext = 'functional_BO_international_taxes_pagination';
+const baseContext = 'functional_BO_international_localization_taxes_sortTaxes';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/05_sortAndPagination.js]: The file is not sync to the `baseContext`.

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/taxes/05_sortTaxes.js [hunk 3]
```diff
@@ -1,121 +0,0 @@
-require('module-alias/register');
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/03_taxes/taxes/05_sortTaxes.js:1]: @boubkerbribri Are you agree with this deletion ?

**Review by Progi1984 — COMMENTED** [review]
@boubkerbribri Thank you for your contribution. I brought to you some comments. Please check it.

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/taxes/05_sortTaxes.js (hunk 3 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/functional/BO/11_international/03_taxes/taxes/05_sortTaxes.js:1]: I added pagination to this and renamed the file

File: tests/UI/campaigns/functional/BO/11_international/01_localization/02_languages/02_CRUDLanguage.js [hunk 4]
```diff
@@ -112,17 +112,29 @@ describe('CRUD language', async () => {
       await expect(numberOfLanguagesAfterCreation).to.be.equal(numberOfLanguages + 1);
     });
 
-    it(`should go to FO and check that '${createLanguageData.name}' exist`, async function () {
-      await testContext.addContextItem(this, 'testIdentifier', 'checkCreatedLanguageFO', baseContext);
+    it('should go to FO', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToFo1', baseContext);
 
-      // View my shop and init pages
+      // View my shop and int pages
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/01_localization/02_languages/02_CRUDLanguage.js]:
```suggestion
      // View my shop and init pages
```

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/05_sortAndPagination.js [hunk 5]
```diff
@@ -12,14 +12,13 @@ const taxesPage = require('@pages/BO/international/taxes');
 // Import test context
 const testContext = require('@utils/testContext');
 
-const baseContext = 'functional_BO_international_taxes_pagination';
+const baseContext = 'functional_BO_international_localization_taxes_sortAndPagination';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/05_sortAndPagination.js]:
```suggestion
const baseContext = 'functional_BO_international_taxes_taxes_sortAndPagination';
```

**Review by Progi1984 — COMMENTED** [review]
@boubkerbribri Thank you for your contribution. I have added two comments to your PR.

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/02_CRUDTaxesInBO.js [hunk 6]
```diff
@@ -15,16 +15,17 @@ const TaxFaker = require('@data/faker/tax');
 // Import test context
 const testContext = require('@utils/testContext');
 
-const baseContext = 'functional_BO_international_localization_taxes_CRUDTax';
+const baseContext = 'functional_BO_international_taxes_taxes_CRUDTax';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/02_CRUDTaxesInBO.js]:
```suggestion
const baseContext = 'functional_BO_international_taxes_taxes_CRUDTaxInBO';
```

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/03_taxesBulkActionsInBO.js [hunk 7]
```diff
@@ -15,7 +15,7 @@ const TaxFaker = require('@data/faker/tax');
 // Import test context
 const testContext = require('@utils/testContext');
 
-const baseContext = 'functional_BO_international_localization_taxes_taxesBulkActions';
+const baseContext = 'functional_BO_international_taxes_taxes_taxesBulkActions';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/03_taxesBulkActionsInBO.js]:
```suggestion
const baseContext = 'functional_BO_international_taxes_taxes_taxesBulkActionsInBO';
```

File: tests/UI/campaigns/functional/BO/11_international/04_translations/01_modifyTranslation.js [hunk 8]
```diff
@@ -15,12 +15,12 @@ const {Languages} = require('@data/demo/languages');
 // Import test context
 const testContext = require('@utils/testContext');
 
-const baseContext = 'functional_BO_international_localization_translations_editTranslation';
+const baseContext = 'functional_BO_international_translations_editTranslation';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/04_translations/01_modifyTranslation.js]:
```suggestion
const baseContext = 'functional_BO_international_translations_modifyTranslation';
```

**Review by Progi1984 — COMMENTED** [review]
@boubkerbribri Some `baseContext` to change

File: tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/02_CRUDTaxesInBO.js [hunk 9]
```diff
@@ -15,16 +15,17 @@ const TaxFaker = require('@data/faker/tax');
 // Import test context
 const testContext = require('@utils/testContext');
 
-const baseContext = 'functional_BO_international_localization_taxes_CRUDTax';
+const baseContext = 'functional_BO_international_taxes_taxes_CRUDTaxInBO';
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/11_international/03_taxes/01_taxes/02_CRUDTaxesInBO.js]:
```suggestion
const baseContext = 'functional_BO_international_taxes_taxes_CRUDTaxesInBO';
```

**Review by Progi1984 — COMMENTED** [review]
@boubkerbribri It seems to remain a last comment ;)

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @boubkerbribri

**Pull request merged**